docs/Hosting on Glitch.md

```diff
@@ -0,0 +1,55 @@
+![Glitch Logo](https://miro.medium.com/max/541/1*qilnXz9CPIMPpr-eN6pNyg.png)
+# 🌥️ Hosting on Glitch.com
+The free hosting service - useful for people that don't want to pay for a host.
+
+Pros: It works surprisingly well and you shouldn't have any major issues if you follow the instructions in this guide correctly.
+
+Cons: You'll have to wipe old logs every now and then when your upper limit of about 200MB is capped. If you're okay with not having old logs, this is the host for you. Remember, its hard to fill up 200MB worth of text so you should be good for a while.
```

Comment:
This is contradicted by the next.  Keep the personal out of it.  If any guidance in decision making is to be offered it should be to discourage use and cast it in light of "these tips work for some people on this non-supported platform."  The only recommended way to host the bot is on a standard windows, linux/unix, or macos environment; all others are discouraged and non-supported.  It is essential to stress that it is non-supported, as future updates will not be tested against it, and it could break on glitch at any point.  The bot is only tested against standard environments (e.g. linux/unix, macos, windows).

Reply:
Lol no. This is a guide on how to use it for people that want to use Glitch. I'm not going to discourage people using a certain host based on someone else's poor opinions. Stay out of this, you've clearly never used Glitch before and clearly have no idea about it.

Reply:
Glitch is for testing, not production. It is annoying to deal with, and not supported for a reason. Nothing wrong with starting this PR, but just keep in mind it probably won't be merged.